test/server/hot_restart_impl_test.cc

```diff
@@ -132,15 +136,15 @@ TEST_P(HotRestartImplAlignmentTest, objectOverlap) {
   std::vector<TestStat> stats;
   for (uint64_t i = 0; i < num_stats_; i++) {
     std::string name = fmt::format("{}zzzzzzzzzzzzzzzzzzzzzzzzzzzz", i)
-                           .substr(0, Stats::RawStatData::maxNameLength());
+                           .substr(0, Stats::RawStatData::maxObjNameLength());
```

Comment:
I think this should stay maxNameLength(), because it's directly creating the stat name